core/block/import/markdown/import_test.go

```diff
@@ -75,6 +76,89 @@ func TestMarkdown_GetSnapshots(t *testing.T) {
 		assert.Nil(t, sn)
 		assert.True(t, err.IsNoObjectToImportError(1))
 	})
+	t.Run("import file with links", func(t *testing.T) {
+		// given
+		converter := newMDConverter(&MockTempDir{})
+		h := &Markdown{blockConverter: converter}
+		p := process.NewProgress(pb.ModelProcess_Import)
+
+		// when
+		sn, err := h.GetSnapshots(context.Background(), &pb.RpcObjectImportRequest{
+			Params: &pb.RpcObjectImportRequestParamsOfMarkdownParams{
+				MarkdownParams: &pb.RpcObjectImportRequestMarkdownParams{Path: []string{"testdata"}},
+			},
+			Type: model.Import_Markdown,
+			Mode: pb.RpcObjectImportRequest_IGNORE_ERRORS,
+		}, p)
+
+		// then
+		assert.Nil(t, err)
+		assert.NotNil(t, sn)
+		assert.Len(t, sn.Snapshots, 4)
+
+		var found bool
+		for _, snapshot := range sn.Snapshots {
+			if snapshot.FileName == "testdata/links.md" {
+				found = true
+				assert.Len(t, snapshot.Snapshot.Data.Blocks, 14)
+				assertLinkBlocks(t, snapshot)
+			}
+		}
+		assert.True(t, found)
+	})
+}
+
+func assertLinkBlocks(t *testing.T, snapshot *common.Snapshot) {
```

Comment:
please, assert whole blocks 

you can use `blockbuilder` to compare two state trees, it ignores blockIds and compares only content and "shape" of document

Reply:
Added